tensorboard/webapp/metrics/views/card_renderer/scalar_card_linked_time_fob_test.ts

```diff
@@ -86,4 +86,41 @@ describe('ScalarLinkedTimeFobController', () => {
       );
     });
   });
+
+  describe('getHighestStep', () => {
+    it('gets the highest step when minMax is in order', () => {
+      const minMax: [number, number] = [0, 2];
+      const fixture = createComponent({minMax});
+
+      const highestStep = fixture.componentInstance.getHighestStep();
+
+      expect(highestStep).toBe(2);
+    });
+    it('gets the highest step when minMax is not in order', () => {
+      const minMax: [number, number] = [2, 0];
+      const fixture = createComponent({minMax});
+
+      const highestStep = fixture.componentInstance.getHighestStep();
+
+      expect(highestStep).toBe(2);
+    });
+  });
+  describe('getLowestStep', () => {
```

Comment:
nit: newline between each "describe" and "it"

Reply:
done.